wasm/section.go

```diff
@@ -9,16 +9,15 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"sort"
 
 	"github.com/go-interpreter/wagon/wasm/internal/readpos"
 	"github.com/go-interpreter/wagon/wasm/leb128"
 )
 
-type sectionInf interface {
+type Section interface {
 	SectionID() SectionID
-	RawSection() *Section
+	GetRawSection() *RawSection
```

Comment:
I would just go with `s/GetRawSection/RawSection/`

Reply:
Not possible, because it's an embedded field in other sections